### PR TITLE
#5588 - Button alignment issue on whats on this weekend or today.

### PR DESCRIPTION
**Pull request by ishouty**
Fix styling issue with buttons on card grid component. By adding margin top .

* Add vertical alignment for button links on card grid
* Update package.json to run scripts for testing
* Start writing some unit tests for CardGrid component
* Added snapshots

### CODE REVIEW

File: common/views/components/CardGrid/CardGrid.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ import { type Book } from '../../../model/books';
 import { type Article } from '../../../model/articles';
 import Space from '../styled/Space';
 import type Moment from 'moment';
+import React from 'react';
```
Comment (gestchild) [inline common/views/components/CardGrid/CardGrid.js:18]: Not that it matters hugely but we usually just import { Fragment } from 'react';

File: common/package.json [hunk 2]
```diff
@@ -2,7 +2,9 @@
   "name": "@weco/common",
   "version": "1.0.0",
   "scripts": {
-    "test": "jest"
+    "test": "jest",
+    "test:watch": "npm run test -- --watchAll",
```
Comment (davidpmccormick) [inline common/package.json:6]: prefer `yarn` for consistency, I think.

**Review by davidpmccormick — COMMENTED** [review]
I feel like some of the tests are maybe slightly superfluous, but that possibly says more about me than about the code!

I think I can just about see the benefit of the snapshot, because it guards us against inadvertently changing how the component looks without checking that is still looks ok (which is how this bug crept in initially).

But to test that the number of links is right, or that they contain text, feels more like a test of whether JavaScript `map` or React's internals are working (which I don't think it should be up to us to do – I'd like to think that they have that covered at their end!).

I flip-flop on this stuff, though – curious to hear what @gestchild and @jamesgorrie think?

**Comment by ishouty** [comment]
> I feel like some of the tests are maybe slightly superfluous, but that possibly says more about me than about the code!
> 
> I think I can just about see the benefit of the snapshot, because it guards us against inadvertently changing how the component looks without checking that is still looks ok (which is how this bug crept in initially).
> 
> But to test that the number of links is right, or that they contain text, feels more like a test of whether JavaScript `map` or React's internals are working (which I don't think it should be up to us to do – I'd like to think that they have that covered at their end!).
> 
> I flip-flop on this stuff, though – curious to hear what @gestchild and @jamesgorrie think?

**Review by jamesgorrie — APPROVED** [review]

**Pull request merged**